src/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@
 
 celeritas_version_to_hex(CELERITAS_VERSION PROJECT_VERSION)
 celeritas_configure_file("celeritas_version.h.in" "celeritas_version.h" @ONLY)
+celeritas_configure_file("corecel/Version.hh.in" "corecel/Version.hh" @ONLY)
```

Comment:
Since these headers now live in corecel, I think the corresponding cmake should as well.